includes/ApplHandlingByStr.php

```diff
@@ -33,6 +34,46 @@ function showStatus( $stat_str )
 		{
 			echo "<td><b>REJECTED</b> by ".$str[0] ."</td>";
 		}
+		else if( $str[1] == 'F' )
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Usage of ELSE IF is discouraged; use ELSEIF instead
* Expected 1 space after IF keyword; 0 found
* Expected 0 spaces after opening bracket; 1 found
* Expected 0 spaces before closing bracket; 1 found
* Expected 1 space after closing parenthesis; found 9

<!-- Comment by [SideCI](https://sideci.com) -->